apps/admin-gui/src/app/core/services/common/notificator.service.ts

```diff
@@ -24,13 +22,33 @@ export class NotificatorService {
   @Output()
   addNotification: EventEmitter<NotificationData> = new EventEmitter<NotificationData>();
 
+  getDefaultActionMessage(): string {
+    if (this.defaultAction === undefined) {
+      this.translate.get('NOTIFICATOR.NOTIFICATION.DEFAULT_ACTION').subscribe(value => {
+        return this.defaultAction = value;
+      });
+    } else {
+      return this.defaultAction;
+    }
+  }
+
+  getDefaultRpcMessage(): string {
+    if (this.defaultRpcMessage === undefined) {
+      this.translate.get('NOTIFICATOR.NOTIFICATION.DEFAULT_RPC_ERROR_MESSAGE').subscribe(value => {
```

Comment:
You should use here synchronous `instant` instead of the async `get`. The return in the lamba doesn't work for the whole method, it is bounded to the given lambda.